testing.py

```diff
@@ -0,0 +1,116 @@
+import os 
+import unittest
+
+from app import app
+
+class testCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
no need to create declare these methods if you don't need them... the methods with only `pass`